site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "setup minikube in China"
+linkTitle: "Setup minikube in China"
+weight: 1
+---
+
+This Tutorial explains how to setup minikube in China.
+
+### Binary download
+
+For more detail please refer to [minikube installation guide]({{<ref "/docs/start">}})
+
+Open a terminal. If on Windows, open a terminal with administrator access. Run:
+
+```shell
+minikube start --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers
+```
+If minikube fails to start, see the [drivers page]({{<ref "/docs/drivers">}}) for help setting up a compatible container or virtual-machine manager.
+
+For users who use the [Hyper-V](https://minikube.sigs.k8s.io/docs/drivers/hyperv/) environment, you should first open the Hyper-V manager to create an external virtual switch. After that, we can use the following commands to create a Hyper-V-based Kubernetes test environment:
+
+```shell
+.\minikube.exe start --image-mirror-country cn \
+    --iso-url=https://kubernetes.oss-cn-hangzhou.aliyuncs.com/minikube/iso/minikube-v1.12.0.iso \
+    --registry-mirror=https://xxxxxx.mirror.aliyuncs.com \
+    --vm-driver="hyperv" \
+    --hyperv-virtual-switch="MinikubeSwitch"
```

Comment:
you can omit the hyperv virtual switch reference here, most windows 10 machines will work without it and our hyperv documentation will help otherwise

Reply:
may be we can point to hyper-v doc somewhere in this doc.

Reply:
Omitted!